src/hooks/useCloudCannonEditor.ts

```diff
@@ -0,0 +1,18 @@
+import { useEffect, useState } from 'react';
+
+export const useCloudCannonEditor = () => {
+  const [isEditable, setIsEditable] = useState(false);
+
+  useEffect(() => {
+    // Check if window.editable exists (CloudCannon specific)
+    // or window.inEditorMode (from our previous setup)
+    if (
+      typeof window !== 'undefined' && 
+      ((window as any).editable || (window as any).inEditorMode)
```

Comment:
As far as I know `window.editable` isn't something we set 🤔You also shouldn't need the `as any`
```suggestion
     window.inEditorMode
```